pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -181,14 +190,8 @@
     <NETCoreApp10Dependency Include="System.Threading.ThreadPool">
       <Version>4.0.10</Version>
     </NETCoreApp10Dependency>
-    <NETCoreApp10Dependency Include="Libuv">
-      <Version>1.9.2</Version>
-    </NETCoreApp10Dependency>
-    <NETCoreApp10Dependency Include="Microsoft.NETCore.Platforms">
-      <Version>1.0.2</Version>
-    </NETCoreApp10Dependency>
-    <NETCoreApp10Dependency Include="System.Net.Http">
-      <Version>4.1.3</Version>
+    <NETCoreApp10Dependency Include="System.Xml.XmlReaderWriter">
```

Comment:
While I'm fine with updating these I don't usually bother doing this update until we go stable.